ui/app/src/views/projects/ProjectTabs.tsx

```diff
@@ -314,6 +299,78 @@ function TabButton({ index, projectName, ...props }: TabButtonProps) {
           />
         </>
       );
+    case secretsTabIndex:
+      return (
+        <>
+          <CRUDButton
+            action="create"
+            scope="Secret"
+            project={projectName}
+            variant="contained"
+            onClick={() => setSecretDrawerOpened(true)}
+            {...props}
+          >
+            Add Secret
+          </CRUDButton>
+          <SecretDrawer
+            secret={{
+              kind: 'Secret',
+              metadata: {
+                name: 'NewSecret',
+                project: projectName,
+              },
+              spec: {
+                basicAuth: {
+                  username: '',
+                  password: '',
+                  passwordFile: '',
+                },
+              },
+            }}
+            isOpen={isSecretDrawerOpened}
+            action="create"
+            isReadonly={isReadonly}
+            onSave={handleSecretCreation}
+            onClose={() => setSecretDrawerOpened(false)}
+          />
+        </>
+      );
+    case variablesTabIndex:
```

Comment:
Did anything change with variables as part of this PR or is the diff just some reorganization / cleanup?

Reply:
Yup, reorg by alpha order